### PR TITLE
Feature/3266 value converters needs to resolve links in the correct culture

### PR DESCRIPTION
**Pull request by MHove89**
**Findings**

```
`Danish node referencing a Danish node from the RTE linkpicker
"/People" before changes "/mennesker" after.

Results before
{
	"sourceId": "gid://Source/bf50bc5a-116e-4a04-b1dc-0bc754e2ab74",
	"id": "gid://Source/bf50bc5a-116e-4a04-b1dc-0bc754e2ab74/Entity/1131-da",
	"type": "contact",
	"originId": "1131-da",
	"originParentId": "1102-en-us",
	"url": "https://localhost:44313/da/kontakt/",
	"redirects": [],
	"properties": {
		"seoMetaDescription": "",
		"keywords": [],
		"umbracoNavihide": false,
		"pageTitle": "Let's have a chat",
		"contactFormHeader": "Send Us A Message",
		"contactIntro": "<p>test <a href=\"/people/\" title=\"Mennesker\" data-anchor=\"#\">Mennesker</a></p>",
		"mapHeader": "You'll find us here",
		"metaData": {
			"name": "Kontakt",
			"culture": "da",
			"domain": null,
			"sortOrder": 4,
			"level": 2,
			"createDate": "2023-06-12T23:10:51",
			"updateDate": "2023-06-17T12:13:07",
			"nodePath": [
				"1102-da",
				"1131-da"
			]
		}
	}
}



Results after
{
	"sourceId": "gid://Source/bf50bc5a-116e-4a04-b1dc-0bc754e2ab74",
	"id": "gid://Source/bf50bc5a-116e-4a04-b1dc-0bc754e2ab74/Entity/1131-da",
	"type": "contact",
	"originId": "1131-da",
	"originParentId": "1102-en-us",
	"url": "https://localhost:44313/da/kontakt/",
	"redirects": [],
	"properties": {
		"seoMetaDescription": "",
		"keywords": [],
		"umbracoNavihide": false,
		"pageTitle": "Let's have a chat",
		"contactFormHeader": "Send Us A Message",
		"contactIntro": "<p>test <a href=\"/da/mennesker/\" title=\"Mennesker\" data-anchor=\"#\">Mennesker</a></p>",
		"mapHeader": "You'll find us here",
		"metaData": {
			"name": "Kontakt",
			"culture": "da",
			"domain": null,
			"sortOrder": 4,
			"level": 2,
			"createDate": "2023-06-12T23:10:51",
			"updateDate": "2023-06-17T12:18:24",
			"nodePath": [
				"1102-da",
				"1131-da"
			]
		}
	}
}


Danish node referencing a Danish node from a RTE component, using link picker in the rte,  in a grid block.
"/da/produkt"
{
	"sourceId": "gid://Source/bf50bc5a-116e-4a04-b1dc-0bc754e2ab74",
	"id": "gid://Source/bf50bc5a-116e-4a04-b1dc-0bc754e2ab74/Entity/1127-da",
	"type": "blog",
	"originId": "1127-da",
	"originParentId": "1102-en-us",
	"url": "https://localhost:44313/da/blog/",
	"redirects": [],
	"properties": {
		"pageTitle": "Behind The Scenes",
		"bodyText": {
			"items": [
				{
					"content": {
						"richText": "<p><a href=\"/da/produkt/\" title=\"Produkt\" data-anchor=\"#\">Produkt</a></p>"
					},
					"contentType": "umbBlockGridDemoRichTextBlock",
					"columnSpan": 12,
					"rowSpan": 1
				}
			]
		},
		"seoMetaDescription": "",
		"keywords": [],
		"umbracoNavihide": false,
		"howManyPostsShouldBeShown": 2,
		"disqusShortname": "",
		"metaData": {
			"name": "Blog",
			"culture": "da",
			"domain": null,
			"sortOrder": 3,
			"level": 2,
			"createDate": "2023-06-12T23:10:51",
			"updateDate": "2023-06-17T20:08:52",
			"nodePath": [
				"1102-da",
				"1127-da"
			]
		}
	}
}
```

### CODE REVIEW

**Comment by emilras** [comment]
Thanks for the notice Mathias! I have removed my self as a reviewer, and leaving it for @jesperweber :)

File: src/Enterspeed.Source.UmbracoCms/Services/DataProperties/DefaultConverters/DefaultMultiUrlPickerPropertyValueConverter.cs [hunk 1]
```diff
@@ -93,8 +90,6 @@ private ObjectEnterspeedProperty ConvertToEnterspeedProperty(Link link, IUmbraco
                 if (link.Udi.EntityType == "document")
                 {
                     var content = context.Content.GetById(link.Udi);
-                    // link.url always contains the url for the default culture (maybe a bug in Umbraco?), so we resolve the url for the correct culture our self
```
[the remark this answers is not in the document]
Comment (MHove89) [reply src/Enterspeed.Source.UmbracoCms/Services/DataProperties/DefaultConverters/DefaultMultiUrlPickerPropertyValueConverter.cs:96]: Seems that setting the variant context fixes the issue :)

[the remark this answers is not in the document]
Comment (jesperweber) [reply src/Enterspeed.Source.UmbracoCms/Services/DataProperties/DefaultConverters/DefaultMultiUrlPickerPropertyValueConverter.cs:96]: Awesome :-)

**Review by jesperweber — DISMISSED** [review]
Looks really great. If you just update the changelog file then we are good to go.

**Comment by MHove89** [comment]
This has now been implemented for Umbraco 8 as well

**Comment by MHove89** [comment]
@jesperweber i just need the last changes approved, then we can merge this task:)

**Review by jesperweber — APPROVED** [review]
Looking good.

**Review by jesperweber — APPROVED** [review]

**Pull request merged**